backend/core/interactem/core/models/operators.py

```diff
@@ -111,6 +111,17 @@ class OperatorParameter(BaseModel):
     value: str | None = None  # Value of the parameter
     options: list[str] | None = None  # List of options for STR_ENUM
 
+class OperatorTagType(str, Enum):
+    FACILITY = "facility"  # For facility-specific resources (like "ncem-4dcamera")
+    CAPABILITY = "capability"  # For general capabilities (like "gpu")
```

Comment:
I am not sure we want to make a distinction? It doesn't seem to be used programmatically, so seems to be more targeted a the user, in which case this information could be convade in the description?

Reply:
OK, so this field can be left out?